lib/hammer_cli_foreman_ansible/associated_ansible_role.rb

```diff
@@ -72,9 +72,7 @@ def ids_to_keep_for_removal(roles)
         next role['id'] if role['directly_assigned']
 
         # Pre-check to force stop the command if we're trying to remove not directly assigned role
-        if role['id'] == @associated_identifier
-          raise ArgumentError, _('Ansible role %s is not assigned directly and cannot be removed.') % role['name']
-        end
+        raise ArgumentError, _('Ansible role %s is not assigned directly and cannot be removed.') % role['name'] if role['id'] == @associated_identifier
```

Comment:
I'm not a fan of this cop because IMHO this is less readable due to its line length, so consider disabling Style/GuardClause.